src/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -5,31 +5,91 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.IO.Tests
 {
-    public class NullCtorTests
+    public class NullTests
     {
+        [Theory]
+        [InlineData(20)]
+        public async static Task TestNullStream_Flush(int times)
+        {
+            for (int i = 0; i < times; i++)
+            {
+                Stream.Null.Flush();
+                await Stream.Null.FlushAsync();
+            }
+        }
+        
+        [Theory]
+        [InlineData(20)]
+        public static void TestNullStream_Dispose(int times)
+        {
+            for (int i = 0; i < times; i++)
+            {
+                Stream.Null.Dispose();
+            }
+        }
+        
+        [Theory]
+        [MemberData(nameof(NullStream_CopyToData))]
+        public async static Task TestNullStream_CopyTo(Stream destination, int bufferSize, CancellationToken cancellationToken)
+        {
+            Stream.Null.CopyTo(destination, bufferSize);
+            await Stream.Null.CopyToAsync(destination, bufferSize, cancellationToken);
+        }
+        
         [Fact]
-        public static async Task TestNullStream()
+        public async static Task TestNullStream_CopyToAsyncValidation()
         {
-            Stream s = Stream.Null;
-
-            s.Flush();
-
-            Assert.Equal(-1, s.ReadByte());
-
-            s.WriteByte(5);
-            int n = s.Read(new byte[2], 0, 2);
-            Assert.Equal(0, n);
-            s.Write(new byte[2], 0, 2);
-
-            Assert.Equal(0, await s.ReadAsync(new byte[2], 0, 2));
-
-            s.Flush();
-            s.Dispose();
+            // Since Stream.Null previously inherited its CopyToAsync
+            // implementation from the base class, which did check its
+            // arguments, we have to make sure it validates them as
+            // well for compat.
```

Comment:
I added it since the reader might be curious why `WriteAsync` / `ReadAsync` were being tested with invalid values, but `CopyToAsync` performed validation.